05-read-write-plot.Rmd

```diff
@@ -150,109 +260,26 @@ raster_filepath = system.file("raster/srtm.tif", package = "spDataLarge")
-                               "rWBclimate", "An access to the World Bank climate data used in the World Bank climate knowledge portal."
-)
-knitr::kable(datapackages, caption = "Selected R packages for spatial data retrieval")
-```
-
-<!-- https://cdn.rawgit.com/Nowosad/Intro_to_spatial_analysis/05676e29/Intro_to_spatial_analysis.html#39 -->
-<!-- Maybe add a section to Data I/O on where and how to retrieve data (with a focus on free data): osmdata (OpenStreetMap; maybe mention TomTom, HERE), Landsat (wrspathrow), Sentinel (mention Python API), AVHRR, RapidEye rgbif, letsR, etc. Of course, point to Transforming science through open data project (https://www.ropensci.org) -->
-<!-- https://github.com/ropensci/GSODR -->
-<!-- https://github.com/lbusett/MODIStsp -->
-<!-- https://github.com/walkerke/tigris -->
-<!-- https://github.com/ropensci/hddtools/ -->
-
-For example, you can get borders of any country with the `ne_countries()` function from the **rnaturalearth** package:
-```{r}
-library(rnaturalearth)
-usa = ne_countries(country = "United States of America") # United States borders
-class(usa)
-```
-
-As a default, this and many other packages output data in the `Spatial*` class. 
-You can easily convert it to the `sf` class with `st_as_sf()`:
-
-```{r}
-usa_sf = st_as_sf(usa)
-```
-
-It is also possible to get a raster data, for example using `getData()`.
-The code below download a series of rasters that contain global monthly precipitation sums in a ten minutes spatial resolution.
-As a result, the new object is a multilayer object of the `RasterStack` class.
-
-```{r}
-library(raster)
-worldclim_prec = getData(name = "worldclim", var = "prec", res = 10)
-class(worldclim_prec)
-```
-
-Finally, many R packages contain example data as R objects or in a raw form.
-Data in native R format could be accessed with the `data()` function.
-For example, you can get a list of dataset in a package, `data(package = "spData")`, or load a dataset by specifying the name of a dataset, `data("cycle_hire", package = "spData")`.
-To access a single raw file from the package, you need to specify the package name and the relative path to the dataset, for example:
-
-```{r}
-world_raw_filepath = system.file("shapes/world.gpkg", package = "spData")
-world_raw = st_read(world_raw_filepath)
-```
-
-More information on getting data using R packages could be found in [section 5.5](https://csgillespie.github.io/efficientR/input-output.html#download) and [section 5.6](https://csgillespie.github.io/efficientR/input-output.html#accessing-data-stored-in-packages) of @gillespie_efficient_2016.
+Please refer to section \@ref(raster-classes) for information on the difference between raster stacks and bricks.
 
 ## Data output (O) {#data-output}
 
+<!--maybe we can come up withan intro which is a bit more compelling-->
```

Comment:
I think we can write an intro which is a bit more compelling. I have to run now, so maybe you can come up with sth. or else I will think about it later on.